staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -101,7 +101,7 @@ func ValidateCustomResourceDefinitionSpec(spec *apiextensions.CustomResourceDefi
 	}
 
 	allErrs = append(allErrs, ValidateCustomResourceDefinitionNames(&spec.Names, fldPath.Child("names"))...)
-	allErrs = append(allErrs, ValidateCustomResourceDefinitionSchema(spec.Validation.JSONSchema, fldPath.Child("Validation.JSONSchema"))...)
+	allErrs = append(allErrs, ValidateCustomResourceDefinitionValidation(&spec.Validation, fldPath.Child("validation"))...)
```

Comment:
this commit "Validate CRD Schema" can be squashed into "Validate CustomResource  …", can't it?